mtlpy/templates/inc/footer.html

```diff
@@ -85,8 +84,7 @@
       }(document, 'script', 'facebook-jssdk'));
     </script>
     {% if GOOGLE_ANALYTICS %}
-    <script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
-    </script>
+    <script src="https://www.google-analytics.com/urchin.js" type="text/javascript"></script>
```

Comment:
I don't know if we want to force HTTPS?
Maybe we could just spread https links and keep http working.